conduct.md

```diff
@@ -11,8 +11,49 @@
 * Attendance
 * Deadlines
 * Communication
-* Participation
-* Decisions
+
+### Participation
+
+Detailed paragraph describing the category. (delete when added paragraph)
+
+* **Acceptable Conduct (A):**
+  * *Members shall*
+    * points
+
+* **Unacceptable Conduct (U):**
+  * *Members shall not*
+    * points
+
+### Decisions
+
+A 3/4th majority of people in a group should agree upon a given decision before it's
+implemented. They should be made with supporting evidence that shows
+that the choice is productive. When unsure about making a decision
+it is a good idea to defer to a technical leader or the professor. Before
+deciding you should reach out to other groups whom the choice will influence.
+Teams will determine how long a decision will take to implement. Group decisions
+won't be foreboding, but should unite the group to solve a problem together!
+
+* **Acceptable Conduct (A):**
+  * *Members shall*
+    * Take into account everyone's input in the decision making process.
+    * Stay flexible and be willing to change if it would be productive.
+    * Report errors or bugs to the appropriate team members.
+    * Record what choice was made and when it was implemented.
+    * Defer to the Professor or student technical leaders when unsure.
+    * Use online polls to reach a group consensus about a decision.
+    * Distribute tasks to all group members.
+    * A new decision should be made when a previous decision cannot be implemented.
+    * When a choice is disputed use a slack poll using the extension Polly.
```

Comment:
There should be a comma after the word disputed.
Remove line 44 for the repetition